com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformOwnershipTests.cs

```diff
@@ -0,0 +1,277 @@
+#if COM_UNITY_MODULES_PHYSICS
```

Comment:
Why does this need to be here, and not elsewhere?

Also, is it needed, given that you added the physics module as a dependency to the package?

Reply:
It is a version define depending upon whether the physics module is included or not when test runner is running:
```
        {
            "name": "com.unity.modules.physics",
            "expression": "",
            "define": "COM_UNITY_MODULES_PHYSICS"
        }
```
So, it isn't really adding it as a dependency but just figuring out if it is available.  There are some tests (i.e. test project minimal) that do not include physics and as such things like Rigidbody won't exist.  Under this scenario and specific to what is being tested, it is actually a requirement now if you include anything from com.unity.modules.physics.
If so, then you need to wrap your integration test with #if COM_UNITY_MODULES_PHYSICS if your test depends upon it or around the areas that have dependencies depending upon the integration test(s) requirements.
_(I actually just found this out by bumping up against that wall in this PR)_